.github/workflows/compilation-push-master.yaml

```diff
@@ -37,8 +45,8 @@ jobs:
         uses: jakejarvis/s3-sync-action@v0.5.1
         with:
           args: --acl public-read --follow-symlinks
-          env:
-            AWS_S3_BUCKET: autoware-modelzoo
-            AWS_ACCESS_KEY_ID: ${{ secrets.MODELZOO_S3_ID }}
-            AWS_SECRET_ACCESS_KEY: ${{ secrets.MODELZOO_S3_KEY }}
-            SOURCE_DIR: '/tmp/artifacts'
+        env:
+          AWS_S3_BUCKET: autoware-modelzoo
+          AWS_ACCESS_KEY_ID: ${{ MODELZOO_S3_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ MODELZOO_S3_KEY }}
```

Comment:
@JWhitleyWork do you know what is the correct syntax and macro to use here?

Reply:
It should be `${ secrets.MODELZOO_S3_ID }` and `${ secrets.MODELZOO_S3_KEY }`. However, I just noticed the following note on the doc pages:

```
Note: With the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.
```

So this won't work here unless we merge it. Please set back to your previous values and I'll merge it now that the job otherwise runs successfully.

Reply:
@JWhitleyWork done